### PR TITLE
*: update metadata for publishing

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -43,3 +52,7 @@ rand = "0.3"
 
 [profile.release]
 debug = true
+
+[badges]
+travis-ci = { repository = "pingcap/grpc-rs" }
+appveyor = { repository = "busyjay/grpc-rs" }
```
Comment (siddontang) [inline Cargo.toml]: can we use pingcap/grpc-rs here？

**Comment by BusyJay** [comment]
PTAL

File: compiler/Cargo.toml [hunk 2]
```diff
@@ -1,8 +1,14 @@
 [package]
 name = "grpcio-compiler"
 version = "0.1.0"
+authors = ["The TiKV Project Developers"]
+license = "Apache-2.0"
+keywords = ["compiler", "grpc", "protobuf"]
+repository = "https://github.com/pingcap/grpc-rs"
+homepage = "https://github.com/pingcap/grpc-rs/compiler"
```
Comment (overvenus) [inline compiler/Cargo.toml]: 404 Not Found

File: grpc-sys/Cargo.toml [hunk 3]
```diff
@@ -1,7 +1,25 @@
 [package]
 name = "grpcio-sys"
 version = "0.1.0"
+authors = ["The TiKV Project Developers"]
+license = "Apache-2.0"
+keywords = ["grpc", "bindings"]
+repository = "https://github.com/pingcap/grpc-rs"
+homepage = "https://github.com/pingcap/grpc-rs/grpc-sys"
```
Comment (overvenus) [inline grpc-sys/Cargo.toml]: 404 Not Found

**Comment by BusyJay** [comment]
@siddontang PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**